web/themes/custom/pineapple/components/_card.twig

```diff
@@ -0,0 +1,18 @@
+{% set card_type = card_type|default('vertical') %}
+{% set attributes = attributes|default(create_attribute()).addClass([
+  'card-container',
+]) %}
+<div {{ attributes }}>
+  <div class="card-image-container">
+    <img src="{{ image_url }}" alt="{{ image_alt_text }}">
+  </div>
+  <div class="card-content u-o-3">
+    <h2 class="page-title"><a href="{{ url }}" rel="bookmark">{{ title }}</a></h2>
+    <p>{{ date_posted }}</p>
+    <p>{{ card_body }}</p>
+    {% include "@components/_button.twig" with {
+      button_url: url,
+      button_text: 'Read Article >'
```

Comment:
Button text should come from the node article teaser template not card because if you were to expand the card to be used by the recipe teaser this text wouldn't make sense.